EIPS/eip-1.md

```diff
@@ -1,101 +1,100 @@
 ---
```

Comment:
Just to not lose track of it, root folder `EIPS` will need to be renamed to `EEPS` and `eip-1.md` will need to be renamed to `eep-1.md`.